test/specs/mainnet/swap/Smoke_CrossChainQuote.spec.js

```diff
@@ -0,0 +1,4195 @@
+      );
+      EstimatedGas_Submit = submissionResponse.estimatedGas;
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        EstimatedGas_Estimate,
+        EstimatedGas_Submit,
+        "The Estimated Gas value is not displayed correctly."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        submissionResponse.estimatedGasPrice._hex,
+        "The value of the estimatedGasPrice field of the Submit Batch Response is not displayed."
+      );
+      EstimatedGasPrice_Submit = submissionResponse.estimatedGasPrice._hex;
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        EstimatedGasPrice_Estimate,
+        EstimatedGasPrice_Submit,
+        "The Estimated Gas Price value is not displayed correctly."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNull(
+        submissionResponse.feeToken,
+        "It is not expected behaviour of the feeToken in the Submit Batch Response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        submissionResponse.feeAmount._hex,
+        "The value of the feeAmount field of the Submit Batch Response is not displayed."
+      );
+      FeeAmount_Submit = submissionResponse.feeAmount._hex;
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        FeeAmount_Estimate,
+        FeeAmount_Submit,
+        "The Fee Amount value is not displayed correctly."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNotEmpty(
+        submissionResponse.feeData,
+        "The value of the feeData field of the Submit Batch Response is not displayed."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNull(
+        submissionResponse.delayedUntil,
+        "It is not expected behaviour of the delayedUntil in the Submit Batch Response."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+  });
+
+  // CROSS CHAIN QUOTES ON BSC NETWORK
+  it("Setup the SDK for Bsc network and perform the cross chain quote action", async () => {
+    // initialize the sdk
+    sdkMainNet = new Sdk(process.env.PRIVATE_KEY, {
+      env: EnvNames.MainNets,
+      networkName: NetworkNames.Bsc,
+    });
+
+    assert.strictEqual(
+      sdkMainNet.state.accountAddress,
+      "0xa5494Ed2eB09F37b4b0526a8e4789565c226C84f",
+      "The EOA Address is not displayed correctly."
+    );
+
+    // Compute the smart wallet address
+    let smartWalletOutput = await sdkMainNet.computeContractAccount();
+    smartWalletAddress = smartWalletOutput.address;
+    console.log("Smart wallet address: ", smartWalletAddress);
+
+    assert.strictEqual(
+      smartWalletAddress,
+      "0x666E17ad27fB620D7519477f3b33d809775d65Fe",
+      "The smart address is not displayed correctly."
```

Comment:
```suggestion
      "The smart wallet  address is not displayed correctly."
```

Please change this everywhere where you have specified "smart address" above as it doesn't make sense

Reply:
Modified the text to "The smart wallet address is not displayed correctly." in all the files.